src/components/ButtonPanel.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div className="calculator-base">
+    <div>
```

Comment:
**[OPTIONAL]**
You could store the button name list in an array, loop over the array containing the button names and render button each time you loop over the button name array, it is much more cleaner than what we have now.